packages/core/src/resolver.js

```diff
@@ -138,9 +138,17 @@ export default class Resolver {
       lastError = e
     }
 
+    const isExternal = resolvedPath.includes('/node_modules/')
+
+    // in dev mode make sure to clear the require cache so after
+    // a dev server restart any changed file is reloaded
+    if (this.options.dev && !isExternal && require.cache[resolvedPath]) {
+      delete require.cache[resolvedPath]
```

Comment:
We have a util (in cjs.js) to recursively cleanup required cache

Reply:
Ah yes, missed that. Using that now